src/utils/get-tag-page-uri-component.js

```diff
@@ -5,7 +5,10 @@ const getTagPageUriComponent = tagPageUri => {
         case 'c#':
             return 'csharp';
         default:
-            return tagPageUri.toLowerCase().replace(/\W/g, '-');
+            return tagPageUri
+                .toLowerCase()
+                .replace(/\W/g, '-')
+                .replace(/--/g, '-');
```

Comment:
Were we running into an issue with `--` before? Also not sure if this is working as expected since `/video/youtube/schema-design-anti-patterns--part-3/` still has a `--`.

Reply:
Oh, for some titles with '-' (for example Anti-Patterns - Part 3) this regex is not enough, 
I will improve to better one `/[-_,]+/g`, for this regex it doesn’t matter how many '-' we will have.